src/dashboards/Map/index.tsx

```diff
@@ -7,26 +7,48 @@ import {
     useWalkTime,
 } from '../../logic'
 
+import MapView from '../../components/Map'
+
+import DepartureTag from './DepartureTag'
 import './styles.scss'
-import MapView from './MapView'
 
 const MapDashboard = ({ history }: Props): JSX.Element => {
     const stopPlacesWithDepartures = useStopPlacesWithDepartures()
     const bikeRentalStations = useBikeRentalStations()
     const walkTimes = useWalkTime(stopPlacesWithDepartures)
-
+    const HEADER_MARGIN = 16
+    //Used to calculate the height of the viewport for the map
+    const headerHeight =
+        (document?.getElementsByClassName('header')[0]?.clientHeight ?? 0) +
+        HEADER_MARGIN
```

Comment:
Litt usikker på denne delen egentlig, hvor lesbart er dette?

Viss dere lurer på hvorfor det ble sånn så er det fordi kart komponenten tar 100% av høyden til parent, og viss parent ikke har noe høyde vises ikke kartet. Tenkte da at siden på kart-visning så er det eneste andre på skjermen headeren kunne vi sette wrapper-diven til 100vh - høyden på header

Reply:
Synes ikke det er det mest "lesbare", men har ingen forslag til hvordan det kan løses bedre 😬 Kanskje @draperunner har noe lure forslag?

Reply:
Går det an å bruke `flex: 1` på <div>-en i staden for? For å få den til å fylle opp ledig plass?

Reply:
Funket ikke å ha `flex: 1`på wrapperene rundt kartet, det funket på den Contrast komponenten men da var det mye annet som ikke helt så ut som det skulle :/